src/components/pages/datasets/pep/PepDetail.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useState } from 'react';
+// import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { dateFormatISO } from 'utils';
+import { ChevronDown, ChevronUp } from 'react-feather';
+import { useParams } from 'react-router-dom';
+import Api from 'api';
+
+
+const PepDetail = () => {
+  const { idp } = useParams();
+  const [data, setData] = useState([]);
+  const [error, setError] = useState('');
+  const [isDataReady, setDataReady] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const { t, i18n } = useTranslation();
+  const fetchData = () => {
+    setLoading(true);
+    Api.get(`pep/${idp}/`)
+      .then((resp) => {
+        setData(resp.data);
+        setDataReady(true);
+      })
+      .catch((err) => {
+        if (err.response?.data?.detail) {
+          setError(err.response.data.detail);
+        }
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  };
+  useEffect(() => {
+    fetchData();
+    console.log(data);
+  }, []);
+  const fromPersonLinksLength = data.from_person_links ? data.from_person_links.length : 0;
+  const toPersonLinksLength = data.to_person_links ? data.to_person_links.length : 0;
```

Comment:
?

Reply:
I need it for separate mapping from & to related_person_links & for considering to output view_more button